### PR TITLE
Run the RSpec test on the current line

### PR DESCRIPTION
**Pull request by teoljungberg**
This is done by passing in the line number to `:Dispatch` either by its
absolute line number(`:10Dispatch`, which will run the test on line 10) or
with the relative line number(`:.Dispatch`, which will run the test on
the current line).

Related #3

### CODE REVIEW

File: plugin/rake.vim [hunk 1]
```diff
@@ -152,7 +152,7 @@ function! s:ProjectionistDetect() abort
     else
       let projections['test/*.rb'] = {'dispatch': ruby + ['-Itest', '{file}']}
     endif
-    let projections['spec/*_spec.rb'].dispatch = s:binstub(b:rake_root, 'rspec') + ['{file}']
+    let projections['spec/*_spec.rb'].dispatch = s:binstub(b:rake_root, 'rspec') + ['{file}`=v:lnum ? ":".v:lnum : ""`']
```
Comment (teoljungberg) [inline plugin/rake.vim]: I'm unsure if there's a better way to do this, pointers appriciated.

**Comment by tpope** [comment]
Can you switch the dispatch projections from `["rspec", "{file}"]` style lists to `"rspec {file}"` style strings? It's a long story, but I'd like to clean that legacy mess up before layering this dispatch.vim trick on top of it.

**Comment by teoljungberg** [comment]
@tpope Sure thing, pushed in 83b653a

**Comment by tpope** [comment]
Can you change "projections['*'].make" and the two `.dispatch` values built on it? Thanks so much!

**Comment by teoljungberg** [comment]
Fixed and force-pushed as 93c58929adce0e7eaad327b1c9b36e9b64dd4a8a, all though - I didn't figure out a way to test it out before pushing.

**Comment by teoljungberg** [comment]
@tpope I noticed this behaviour change for `:.Focus` after this PR.

Previously:

```
:.Focus
:18ProjectDo Dispatch bundle exec rspec path/to/some_spec.rb
```

After:

```
:.Focus
:18ProjectDo Dispatch bundle exec rspec /Users/teo.ljungberg/src/klarna/team-panda/Projects/not_necessarily_refunds/spec/lib/not_necessarily_re
funds/app_spec.rb:1818
```

I couldn't track down where the actual behaviour change is, and couldn't formulate a description well enough to open an issue.
And guidance?

**Comment by tpope** [comment]
The duplication at the end was a dispatch.vim bug I just fixed. The remaining two line numbers are a little gross, but accurately reflect how the command will be executed. This would probably be a good time to clean that up by replacing `:ProductDo Dispatch` with `-dir=...` in Projectionist.

**Pull request merged**